OrbitLinuxTracing/PerfEventReaders.h

```diff
@@ -15,6 +15,9 @@ pid_t ReadSampleRecordPid(PerfEventRingBuffer* ring_buffer);
 std::unique_ptr<SamplePerfEvent> ConsumeSamplePerfEvent(
     PerfEventRingBuffer* ring_buffer, const perf_event_header& header);
 
+std::unique_ptr<CallchainSamplePerfEvent> ConsumeSampleCallchainPerfEvent(
```

Comment:
Ah, I think this function requires renaming as well to `ConsumeCallchainSamplePerfEvent`.